cpp/src/parquet/test_encryption_util.h

```diff
@@ -57,6 +57,7 @@ const char kFooterEncryptionKey[] = "0123456789012345";  // 128bit/16
 const char kColumnEncryptionKey1[] = "1234567890123450";
 const char kColumnEncryptionKey2[] = "1234567890123451";
 const char kFileName[] = "tester";
+const char kTempDir[] = "/tmp";
```

Comment:
This is not very customary on Windows. Can you use a cross-platform facility? For example, we have a facility to create temporary directories:
https://github.com/apache/arrow/blob/master/cpp/src/arrow/util/io_util.h#L305